diff -Naur node_modules/pouchdb-live-find/lib/index.js node_modules_2/pouchdb-live-find/lib/index.js
--- node_modules/pouchdb-live-find/lib/index.js	2019-10-19 23:10:28.099000000 +0800
+++ node_modules_2/pouchdb-live-find/lib/index.js	2019-11-29 00:36:01.195818570 +0800
@@ -46,11 +46,12 @@
     sortFn = helpers.createFieldSorter(sort);
   }
   var skip = parseInt(requestDef.skip, 10) || 0;
-  var limit = parseInt(requestDef.limit, 10) || 0;
+  var limit = parseInt(requestDef.limit, 10) || 25;
   var findRequest = {
     selector: selector,
     // sort: sort,
-    fields: fields
+    fields: fields,
+    limit: limit
   };
 
   var ready = db.find(findRequest)
@@ -276,4 +277,4 @@
 
 if (typeof window !== 'undefined' && window.PouchDB) {
   window.PouchDB.plugin(exports);
-}
\ 文件尾没有换行符
+}
diff -Naur node_modules/pouch-vue/src/index.js node_modules_2/pouch-vue/src/index.js
--- node_modules/pouch-vue/src/index.js	2019-10-19 23:10:28.100000000 +0800
+++ node_modules_2/pouch-vue/src/index.js	2019-11-29 00:35:31.790157369 +0800
@@ -324,7 +324,7 @@
                             });
                         });
                     }
-                    return fetchSession();
+                    return fetchSession(databases[db]);
                 },
 
                 sync(localDB, remoteDB=defaultDB, options = {}) {
